src/devices_models/devices/electric_loads.jl

```diff
@@ -153,32 +151,31 @@ function DeviceRangeConstraintSpec(
 end
 
 ########################## Addition to the nodal balances ##################################
-
 function NodalExpressionSpec(
     ::Type{T},
-    ::Type{<:PM.AbstractPowerModel},
+    parameter::ReactivePowerTimeSeries,
     use_forecasts::Bool,
 ) where {T <: PSY.ElectricLoad}
     return NodalExpressionSpec(
-        "max_active_power",
-        "Q",
+        parameter,
+        T,
         use_forecasts ? x -> PSY.get_max_reactive_power(x) : x -> PSY.get_reactive_power(x),
         -1.0,
-        T,
+        :nodal_balance_reactive,
```

Comment:
Do you plan to make changes to this free-form symbol?

Reply:
This will change in an upcoming PR to refactor expression containers.